scripts/databricks_install.py

```diff
@@ -0,0 +1,255 @@
+#!/usr/bin/env python
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# This script installs Recommenders/reco_utils as a library into Databricks
+
+import argparse
+import textwrap
+import os
+
+import shutil
+import sys
+from urllib.request import urlretrieve
+
+## requires databricks-cli to be installed:
+## and requires authentication to be configured
+from databricks_cli.configure.provider import ProfileConfigProvider
+from databricks_cli.configure.config import _get_api_client
+from databricks_cli.clusters.api import ClusterApi
+from databricks_cli.dbfs.api import DbfsApi
+from databricks_cli.libraries.api import LibrariesApi
+
+from requests.exceptions import HTTPError
+
+CLUSTER_NOT_FOUND_MSG = """
+    Cannot find the target cluster {}. Please check if you entered the valid id. 
+    Cluster id can be found by running 'databricks clusters list', which returns a table formatted as:
+
+    <CLUSTER_ID>\t<CLUSTER_NAME>\t<STATUS>
+    """
+
+CLUSTER_NOT_RUNNING_MSG = """
+    Cluster {0} found, but it is not running. Status={1}
+    You can start the cluster with 'databricks clusters start --cluster-id {0}'.
+    Then, check the cluster status by using 'databricks clusters list' and
+    re-try installation once the status becomes 'RUNNING'.
+    """
+""
+
+## Variables for operationalization:
+COSMOSDB_JAR_FILE_OPTIONS = {
```

Comment:
in the near future we would need MMLSpark, we can add it here, or if you prefer to wait until the Ligthgbm work is done, we have to remember to add it later

Reply:
will be happy to go ahead and add it now since I'm doing a lot with it.